src/main/resources/assets/computercraft/lua/rom/programs/delete.lua

```diff
@@ -6,12 +6,20 @@ if args.n < 1 then
 end
 
 for i = 1, args.n do
-    local files = fs.find(shell.resolve(args[i]))
+    local resolvedPath = shell.resolve(args[i])
+    local files = fs.find(resolvedPath)
     if #files > 0 then
         for _, file in ipairs(files) do
-            local ok, err = pcall(fs.delete, file)
-            if not ok then
-                printError((err:gsub("^pcall: ", "")))
+            if file ~= resolvedPath and fs.isReadOnly(file) then
+                print("Skipping read only " .. file)
+            elseif fs.isMountPoint(file) then
+                print("Skipping seperate mount " .. file)
+                print("To delete it's contents run rm " .. fs.combine(file, "*"))
```

Comment:
`its` :p.

You may also want to do `"rm /%s"` - otherwise this'll be wrong when running from outside `/`.